src/StreamJsonRpc.Tests/JsonMessageFormatterTests.cs

```diff
@@ -14,4 +14,11 @@ public JsonMessageFormatterTests(ITestOutputHelper logger)
         : base(logger)
     {
     }
+
+    [Fact]
+    public void DefaultEncodingLacksPreamble()
```

Comment:
Is there any way to run a test that uses the formatter and checks for the lack of a preamble rather than checking the private field directly?

Reply:
It isn't checking a private field. It's checking the public property. But testing that the Encoding property is actually the one used to format the message is a good idea. I'll add that.

Reply:
Yeah, I goofed and misread the `private Encoding encoding` line thinking `Encoding` was the name...